API/application/iotjs.py

```diff
@@ -172,6 +181,11 @@ def build(self, device):
         self.__apply_heap_patches(device)
         self.__apply_stack_patches(device)
 
+        if device.get_type() == 'artik053':
+            coverage_enabled, _, _, _ = device.coverage_info()
+            if coverage_enabled:
+                self.__apply_coverage_patches(device)
```

Comment:
What about having an `is_coverity_enabled` property for `device`?  That property could check the code measurement support by checking the device type and the internal coverage member.